node/resolvers/index.ts

```diff
@@ -879,6 +880,7 @@ export const resolvers = {
           lastUpdate,
           updateHistory,
           status,
+          expirationDate: expirationDate ?? existingQuote.expirationDate,
```

Comment:
We should check if the user has `edit-quotes-all` or `edit-quotes-organization` permissions before allowing them to change the expiration